src/jsx/modal.jsx

```diff
@@ -11,25 +11,36 @@ class ModalMessage extends Component {
         this.state = {
```

Comment:
From our opens source docu:

```
Do I need to add a license header to every source file?

No. In fact, we discourage it because it blows up file sizes, requires some build checking/pre-processing, and sometimes leads to situations like this. It's also not needed for the licenses we use. Having one LICENSE file in the repository is enough.
```